Update award.md - adding a list of 2025 winners

ctivir opened the pull request:
Update award.md - adding a list of 2025 winners

ctivir committed:
Update award.md - adding a list of 2025 winners 

Update award.md - adding a list of 2025 winners

Mariatta commented on docs/global/award.md:
```diff
@@ -61,6 +61,13 @@ if personal preferences change.
+* [Katie McLaughli](https://www.linkedin.com/in/glasnt/)
+* [Karolina Surma](https://www.linkedin.com/in/karolina-surma-950452b7/)
+* [Cecília Tivir] (https://www.linkedin.com/in/ctivir/)
+*
```
Anwesha Das should be added as well.

Mariatta commented on docs/global/award.md:
```diff
@@ -61,6 +61,13 @@ if personal preferences change.
 
+### 2025
+
+* [Katie McLaughli](https://www.linkedin.com/in/glasnt/)
```
Katie McLaughlin

ctivir committed:
Update award.md

Add Anwesha Das to the list
Correct typo on Katie's surname 
Add check out blog's link

Mariatta reviewed (approved):
Thank you.
